bin/glite-info-update-endpoints

```diff
@@ -49,72 +66,68 @@ def parse_args():
             usage()
             sys.exit()
         elif opt == "-v" or opt == "--verbose":
-            log.setLevel(log.debug)
+            LOG.setLevel(LOG.debug)
```

Comment:
this should be LOG.setLevel(logging.DEBUG) otherwise you have:

bash-4.2# ./glite-info-update-endpoints -v
Traceback (most recent call last):
  File "./glite-info-update-endpoints", line 229, in <module>
    parse_args()
  File "./glite-info-update-endpoints", line 67, in parse_args
    LOG.setLevel(LOG.debug)
  File "/usr/lib64/python2.7/logging/__init__.py", line 1125, in setLevel
    self.level = _checkLevel(level)
  File "/usr/lib64/python2.7/logging/__init__.py", line 190, in _checkLevel
    raise TypeError("Level not an integer or a valid string: %r" % level)
TypeError: Level not an integer or a valid string: <bound method RootLogger.debug of <logging.RootLogger object at 0x7f2f23c24410>>

could be that it was not working also before..

Reply:
Maybe you are not aware: there is the nice `Insert a suggestion` feature for this (the +/i icon) to have quick changes.

Reply:
I included this but for next time it could be nice to use suggestions

Reply:
sorry I will use the Insert suggestion next time!

Reply:
No worries, it was just to ensure you were aware.